appengine/test/tc_base_image.rb

```diff
@@ -0,0 +1,18 @@
+require "minitest/autorun"
+require "#{::File.dirname(__FILE__)}/test_helper.rb"
+
+
+# Basic tests on the content of the base image
+
+class TestBaseImage < ::Minitest::Test
+
+  HELPER = TestHelper.new(verbose: ::ENV['VERBOSE'])
```

Comment:
Changed the helper methods to a mixin in https://github.com/GoogleCloudPlatform/ruby-docker/pull/24